APP/src/views/Event.js

```diff
@@ -536,6 +537,11 @@ const Tables = () => {
   const handleChange = (event, newValue) => {
     history.push({ hash: tabDict[newValue] });
     setTabIndex(newValue);
+    if (tabDict[newValue] == "match-scouting"){
```

Comment:
Why would you ever want to turn off snowflakes?

Reply:
It messes with the QR Codes